commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/User.kt

```diff
@@ -1,3 +1,3 @@
 package ebi.ac.uk.model
 
-data class User(val id: Long, val email: String, val secretKey: String)
+data class User(val id: Long, val email: String, val secretKey: String, val fullName: String? = null)
```

Comment:
so full name is declared as nullable but then latter in `RtNotificationSubscriber` we call ` source.user.fullName!!`